.storybook/time-machine.js

```diff
@@ -6,8 +6,8 @@ timemachine is a node module which overrides the system time for a repo. There a
 // Sets the date and time whilst leaving the clock ticking from that point onwards.
 export const startTimeMachine = (tick = true) => {
   timemachine.config({
-    dateString: 'Friday, 9 August 2019 14:04:14 GMT',
-    timestamp: 1565359454,
+    dateString: 'Sunday, 16 February 2020 17:00:00 GMT',
```

Comment:
Ah I see what you were asking before! Does this change a lot of snapshots in Chromatic?

Reply:
Not a lot, but it does change a couple of snapshots. I can revert this line before merging to avoid this :)